services/neo4j/init/init.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+PATH=/opt/neo4j-community-3.3.9/bin:$PATH
```

Comment:
Why version is hardcoded here? Can we use env-variable defined in Docker file here?